classes/core/PKPContainer.php

```diff
@@ -235,7 +235,7 @@ protected function loadConfiguration()
             'driver' => 'database',
             'table' => 'jobs',
             'queue' => 'default',
-            'retry_after' => 240,
+            'retry_after' => 610,
```

Comment:
I left a suggestion regarding these defaults. I think we shouldn't have them, because each job might have a different need, and there's no size fits all... But I don't have anything against increasing the timeout limits/retry delay/etc 😁